### DISCUSSION TITLE
关于聊天历史的长时记忆

### DISCUSSION THREAD

**Discussion by nUser0** [wenda-LLM/wenda#116] [General]
如果可以把聊天历史自动转化到知识库里，是不是就能实现对聊天历史的长时记忆了呢？🤔🤔

**Comment by l15y** [comment]
这个主意很好，但是现在的知识库检索效果不好。等知识图谱模式做好，可以做实时构建图谱

**Reply by nUser0** [reply]
对知识库的检索，有没有可能仿照训练语言模型的方法，以知识库的文本为材料实时训练一个微型的模型？
——只是一个外行的想法……

**Reply by l15y** [reply]
成本太高
